data/extensions/Spells/Commands/KickPlayerCommand.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using NeoServer.Game.Combat.Spells;
+using NeoServer.Game.Common;
+using NeoServer.Game.Common.Contracts.Creatures;
+using NeoServer.Server.Common.Contracts;
+using NeoServer.Server.Helpers;
+
+namespace NeoServer.Extensions.Spells.Commands;
+
+public class KickPlayerCommand : CommandSpell
+{
+    public override bool OnCast(ICombatActor actor, string words, out InvalidOperation error)
+    {
+        var commands = words.Split("/kick");
+
+        if (string.IsNullOrWhiteSpace(commands[1]))
+        {
+            error = InvalidOperation.NotPossible;
+            return false;
+        }
+
+        var ctx  = IoC.GetInstance<IGameCreatureManager>();
+        
+        if (!ctx.TryGetPlayer(commands[1], out var player))
+        {
+            error = InvalidOperation.NotPossible;
+            return false;
+        }
+
+        if (player is null || player.CreatureId == actor.CreatureId)
+        {
+            error = InvalidOperation.NotPossible;
+            return false;
+        }
+        
+        ctx.RemovePlayer(player);
```

Comment:
maybe using the Logout method from the Player is better. You don't need to get the Player from GameCreatureManager. the actor parameter is already a IPlayer, just need to cast.